packages/odyssey-react/src/components/Banner/Banner.module.scss

```diff
@@ -52,8 +51,8 @@
 .dismiss {
   position: absolute;
   inset-block-start: 50%;
-  inset-inline-end: var(--SpaceRemM);
-  margin-inline-start: var(--SpaceRemM);
+  inset-inline-end: var(--DismissInsetInlineEnd);
+  margin-inline-start: var(--DismissMarginInlineStart);
```

Comment:
should we consider naming this `DismissButton` to better distinguish from `Dismissiable` variant below?

Reply:
I'm good with that